hack/arktos-cni.rc

```diff
@@ -66,6 +66,12 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+install_cni_mizar() {
+    sudo rm -rf /opt/cni/bin/
```

Comment:
this function name seems not to reflect what it really does. please consider re-name it.

Reply:
Vinay suggested merge first and we will update this later.